packages/lesswrong/server/scripts/algoliaConfigureIndexes.ts

```diff
@@ -148,9 +139,45 @@ export const algoliaConfigureIndexes = async () => {
     ],
     distinct: false,
     attributesToSnippet: isEAForum ? ['description:20'] : ['description:10'],
-    advancedSyntax: true
-  });
-  
+    advancedSyntax: true,
+  },
+};
+
+export const algoliaConfigureIndexes = async () => {
+  let client = getAlgoliaAdminClient();
+  if (!client) {
+    console.error("Could not get Algolia admin client."); //eslint-disable-line no-console
+    return;
+  }
+
+  console.log("Configuring Algolia indexes"); //eslint-disable-line no-console
+
+  for (const collectionName of algoliaIndexedCollectionNames) {
+    console.log(`...Configuring ${collectionName}`); //eslint-disable-line no-console
+    const index = client.initIndex(getAlgoliaIndexName(collectionName));
+
+    const replicas = isEAForum
+      ? getAlgoliaReplicasForCollection(collectionName)
+      : [];
+    const settings = algoliaIndexSettings[collectionName];
+    await algoliaSetIndexSettingsAndWait(index, {
+      ...settings,
+      replicas: replicas.map((replica) => replica.indexName),
+    });
+
+    for (const {indexName, rankings} of replicas) {
+      console.log(`......Replica ${indexName}`); //eslint-disable-line no-console
+      const replica = client.initIndex(indexName);
+      await algoliaSetIndexSettingsAndWait(replica, {
+        ...settings,
+        ranking: [
+          ...rankings,
+          ...settings.ranking ?? [],
+        ],
+      });
+    }
+  }
```

Comment:
I think it would be helpful to add some comments here, in particular explaining the replicas thing.